docs/tutorials/nb_shap_feature_elimination.ipynb

```diff
@@ -416,24 +240,95 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
+   "execution_count": 22,
    "metadata": {},
    "outputs": [
     {
+     "output_type": "execute_result",
      "data": {
       "text/plain": [
-       "['f14', 'f16', 'f19', 'f15', 'f1_categorical', 'f9', 'f5', 'f20', 'f8']"
+       "['f10',\n",
+       " 'f19',\n",
+       " 'f1_categorical',\n",
+       " 'f12',\n",
+       " 'f9',\n",
+       " 'f15',\n",
+       " 'f8',\n",
+       " 'f20',\n",
+       " 'f14',\n",
+       " 'f16',\n",
+       " 'f5']"
       ]
      },
-     "execution_count": 7,
      "metadata": {},
-     "output_type": "execute_result"
+     "execution_count": 22
     }
    ],
    "source": [
     "shap_elimination.get_reduced_features_set(num_features=11)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also provide a list of features that sholud not be eliminated. \n",
```

Comment:
You can add one more cell with a headline, to distinguish it from the rest

Reply:
Since, this is just a sub feature within SHAPRFECV, a separate headline may not be necessary.What do you think ?

Reply:
Small headline h3 would be sufficient 

### Headline3

Reply:
Done.